README.md

```diff
@@ -5,208 +5,262 @@ This is the official repository for the [Scala Programming Language](http://www.
+ <img src="https://avatars.githubusercontent.com/Ichoran"       height="50px" title="Rex Kerr"/>             | [`@Ichoran`](https://github.com/Ichoran)             | collections library, performance              |
+ <img src="https://avatars.githubusercontent.com/lrytz"         height="50px" title="Lukas Rytz"/>           | [`@lrytz`](https://github.com/lrytz)                 | optimizer, named & default arguments              |
+ <img src="https://avatars.githubusercontent.com/VladUreche"    height="50px" title="Vlad Ureche"/>          | [`@VladUreche`](https://github.com/VladUreche)       | specialization, Scaladoc tool |
+ <img src="https://avatars.githubusercontent.com/densh"         height="50px" title="Denys Shabalin"/>       | [`@densh`](https://github.com/densh)                 | quasiquotes, parser, string interpolators, macros in standard library |
+ <img src="https://avatars.githubusercontent.com/xeno-by"       height="50px" title="Eugene Burmako"/>       | [`@xeno-by`](https://github.com/xeno-by)             | macros and reflection |
+ <img src="https://avatars.githubusercontent.com/heathermiller" height="50px" title="Heather Miller"/>      | [`@heathermiller`](https://github.com/heathermiller)  | documentation |
+ <img src="https://avatars.githubusercontent.com/dickwall"      height="50px" title="Dick Wall"/>            | [`@dickwall`](https://github.com/dickwall)           | process & community, documentation |
+ <img src="https://avatars.githubusercontent.com/dragos"        height="50px" title="Iulian Dragos"/>        | [`@dragos`](https://github.com/dragos)               | specialization, back end |
+ <img src="https://avatars.githubusercontent.com/axel22"        height="50px" title="Aleksandr Prokopec"/>   | [`@axel22`](https://github.com/axel22)               | collections, concurrency, specialization |
+ <img src="https://avatars.githubusercontent.com/janekdb"       height="50px" title="Janek Bogucki"/>        | [`@janekdb`](https://github.com/janekdb)             | documentation |
 
 P.S.: If you have some spare time to help out around here, we would be delighted to add your name to this list!
 
-# Handy Links
-  - [A wealth of documentation](http://docs.scala-lang.org)
-  - [mailing lists](http://www.scala-lang.org/community/)
-  - [Gitter room for Scala contributors](https://gitter.im/scala/contributors)
-  - [Scala CI](https://scala-ci.typesafe.com/)
-  - download the latest nightlies:
-      - [2.11.x](http://www.scala-lang.org/files/archive/nightly/2.11.x/)
-      - [2.12.x](http://www.scala-lang.org/files/archive/nightly/2.12.x/)
-
 # Repository structure
 
 ```
 scala/
-+--build.xml                 The main Ant build script, see also under src/build.
-+--pull-binary-libs.sh       Pulls binary artifacts from remote repository.
++--build.sbt                 The main sbt build script
++--build.xml                 The deprecated Ant build script, see also under src/build.
```

Comment:
If `src/build` is mentioned here then it should also be explained what the contents are and why a contributor might be interested.